packages/fxa-content-server/tests/functional/bounced_email.js

```diff
@@ -14,7 +14,7 @@ let deliveredEmail;
 const PASSWORD = 'password12345678';
 const ENTER_EMAIL_URL = `${
   intern._config.fxaContentRoot
-}?context=fx_desktop_v3&service=sync&automatedBrowser=true&forceAboutAccounts=true&forceUA=${encodeURIComponent(
+}?context=fx_desktop_v3&service=sync&automatedBrowser=true&forceUA=${encodeURIComponent(
   uaStrings.desktop_firefox_58
```

Comment:
should we update this UA?

Reply:
I'm on the fence, if we update the ua string to fx 71 everywhere, we have to hook up WebChannel responses for fxaccounts:fxa_status, but maybe that's a good idea anyways, hook up a default response if one isn't specified.

I'll do that in a separate PR. 👍

Reply:
I have a branch to add default handlers, but have too many PRs open already. I'll open the PR once some of these land.